src/runtime/vendor/github.com/kata-containers/govmm/qemu/qemu.go

```diff
@@ -2440,6 +2440,9 @@ type Config struct {
 	// CPUModel is the CPU model to be used by qemu.
 	CPUModel string
 
+	// SandboxOn is the qemu function which enables the seccomp feature
+	SandboxOn string
+
```

Comment:
Hey, this is not the way to update the vendored code.
You should submit a patch to https://github.com/kata-containers/govmm and then follow something like what has been done on 8385ff9554188d798fba07297240b7cb0feec251.

Reply:
Thank you @fidencio for the comments. I raised https://github.com/kata-containers/govmm/pull/186 against govmm repo . 
After that PR is accepted I will be back here to continue .

Reply:
Amazing, thanks @LiangZhou-CTY!
I started to review your PR there. :-)